app/concerns/wpcc/salary_threshold.rb

```diff
@@ -0,0 +1,9 @@
+module Wpcc
+  module SalaryThreshold
+    extend ActiveSupport::Concern
+
+    def load_config(file:)
+      YAML.load_file(Wpcc::Engine.root.join('config', "#{file}.yml"))
+    end
+  end
+end
```

Comment:
@tomas-stefano this extendable module allows for code reusability. See below for files which can benefit from this approach.  

```
file: app/models/wpcc/contribution_calculator.rb

THRESHOLDS_FILE = Wpcc::Engine.root.join('config', 'earning_thresholds.yml')
THRESHOLDS = YAML.load_file(THRESHOLDS_FILE)
```

```
# improvement

THRESHOLDS = load_file(file: 'earning_thresholds')
```